packages/vm/src/cache.rs

```diff
@@ -28,7 +28,7 @@ pub struct CosmCache<S: Storage + 'static, A: Api + 'static, Q: Querier + 'stati
     wasm_path: PathBuf,
     supported_features: HashSet<String>,
     modules: FileSystemCache,
-    instances: Option<LruCache<Checksum, wasmer_runtime_core::Instance>>,
+    instances: Option<LruCache<Checksum, Box<wasmer_runtime_core::Instance>>>,
```

Comment:
We don't really have to keep these boxed, do we? although i guess in a running node this saves moving the instance into and out of the heap all the time?

Reply:
At some point you were worried about the additional heap allocation. If we do it only once per module instantiation, it is probably negligable compared to to everything else that needs to happen during the instantiation.

Reply:
yeah, sounds good